### DISCUSSION TITLE
Stretch and shrink column header size

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6811] [Ideas]
### Is your feature request related to a problem? Please describe

it does not exist a way to change the column names list

### Describe the solution you'd like

![OpenToonz_1j9hDqxreI](https://github.com/opentoonz/opentoonz/assets/36406840/4721b625-004d-4166-aeac-c8eff01e685d)

just be able to do it

**Comment by RodneyBaker** [comment]
You mean to be able to adjust the width of that header?

This is likely part of a wider resizing that is needed for all columns/rows.
Ideally, it would be like Microsoft Excel where we could drag to change the size or double click to have the cell/header take on the size of the largest entry inside the cells.
This would surely require an extensive rewite of the code however.
